src/btree/bt_curnext.c

```diff
@@ -738,15 +737,12 @@ __wt_btcur_next(WT_CURSOR_BTREE *cbt, bool truncating)
 
 err:
     if (total_skipped < 100) {
-        WT_STAT_CONN_INCR(session, cursor_next_skip_lt_100);
-        WT_STAT_DATA_INCR(session, cursor_next_skip_lt_100);
+        WT_STAT_CONN_DATA_INCR(session, cursor_next_skip_lt_100);
     } else {
-        WT_STAT_CONN_INCR(session, cursor_next_skip_ge_100);
-        WT_STAT_DATA_INCR(session, cursor_next_skip_ge_100);
+        WT_STAT_CONN_DATA_INCR(session, cursor_next_skip_ge_100);
```

Comment:
Both of these now should have the enclosing curly braces removed. And these uses are exactly the issue Alex refers to below.